_studio/shared/umc/core/vm/src/vm_sys_info_linux32.c

```diff
@@ -21,12 +21,8 @@
 #if defined(LINUX32)
 
 #include "vm_sys_info.h"
```

Comment:
Could you, please, modify:
`Copyright (c) 2017 Intel Corporation`
to
`Copyright (c) 2017-2019 Intel Corporation`
on the very top to unlock our CI?